src/shared/const.ts

```diff
@@ -0,0 +1,58 @@
+export enum pagesHash {
+  changePassword= '#changePassword',
+  changeProfile = '#changeProfile',
+  login = '#login',
+  registration = '#registration',
+  profile = '#profile',
+}
+
+export enum RouterLinks {
+  LOGIN= '/#login',
+  REGISTRATION = '/#registration',
+  CHAT = '/',
+  PROFILE = '/#profile',
+  CHANGE_PROFILE = '/#changeProfile',
+  CHANGE_PASSWORD = '/#changePassword'
+}
+export enum RouterLinksName {
+  LOGIN= 'Войти',
+  EXIT= 'Выйти',
+  REGISTRATION = 'Создать аккаунт',
+  CHAT = 'К чатам',
+  PROFILE = 'Профиль',
+  PROFILE_BACK = 'Вернуться в профиль',
+  CHANGE_PROFILE = 'Изменить профиль',
+  CHANGE_PASSWORD = 'Изменить пароль'
+}
+export enum InputName {
+  LOGIN = 'login',
+  PASSWORD = 'password',
+  EMAIL = 'email',
+  FIRST_NAME ='first_name',
+  SECOND_NAME = 'second_name',
+  PHONE = 'phone',
+  CONFIRM = 'confirm',
+  DISPLAY_NAME = 'display_name',
+  OLD_PASSWORD = 'oldPassword',
+  NEW_PASSWORD = 'newPassword',
+  MESSAGE = 'message',
+}
+export enum InputLabel {
+  LOGIN = 'Логин',
+  PASSWORD = 'Пароль',
+  EMAIL = 'Почта',
+  FIRST_NAME ='Имя',
+  SECOND_NAME = 'Фамилия',
+  PHONE = 'Телефон',
+  CONFIRM = 'Пароль еще раз',
+  CONFIRM_NEW = 'Повторите новый пароль',
+  DISPLAY_NAME = 'Имя в чате',
+  OLD_PASSWORD = 'Старый пароль',
+  NEW_PASSWORD = 'Новый пароль',
+}
+
+export enum BUTTON_NAME {
+  LOGIN = 'Войти',
+  REGISTRATION = 'Создать аккаунт',
+  SAVE = 'Сохранить'
```

Comment:
```suggestion
  REGISTRATION = 'Создать аккаунт',
  SAVE = 'Сохранить',
```
Тоже стоит привести к одному виду. Вообще лучше ставить всегда `,` здесь, потому что это положительно сказывается на диффах в системах контроля версий